.github/workflows/tests.yml

```diff
@@ -35,3 +35,25 @@ jobs:
       - run: npm install
       - run: npm run build --if-present
       - run: npm run test
+
+      - name: Create Github URL
+        run: |
+          API_HOST=https://api.github.com
+          OWNER=$(echo $GITHUB_REPOSITORY | cut -d'/' -f1)
+          REPO=$(echo $GITHUB_REPOSITORY | cut -d'/' -f2)
+          # Check out the PR branch
+          git checkout $GITHUB_HEAD_REF
+          # Get the commit ID of the last commit
+          COMMIT_ID=$(git rev-parse HEAD)
+          echo "Last commit ID of PR: $COMMIT_ID"
+          REF=$GITHUB_SHA
```

Comment:
We can directly use $GITHUB_SHA as commit ID?

Reply:
done